### PR TITLE
ndctl: add page

### PR DESCRIPTION
**Pull request by stellarhopper**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/ndctl.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Change the mode of a namespace to 'raw':
+
+`ndctl create-namespace --reconfigure={{namespaceX.Y}} --mode={{raw}}`
+
+- Check (and repair) a sector mode namespace for consistency:
+
+`ndctl check-namespace --repair {{namespaceX.Y}}`
+
+- List all namespaces, regions, buses, including disabled ones:
```
Comment (sbrl) [inline pages/linux/ndctl.md]: Perhaps ...`regions, and buses - including`... would read better?

File: pages/linux/ndctl.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`ndctl list --namespaces --regions --buses --idle`
+
+- List a specific namespace, include lots of additional information:
```
Comment (sbrl) [inline pages/linux/ndctl.md]: Perhaps ....`namespace and include`.... would read better?

File: pages/linux/ndctl.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+`ndctl list -vvv --namespace={{namespaceX.Y}}`
+
+- Run a monitor to watch for SMART health events for NVDIMMs on bus 'ACPI.NFIT':
```
Comment (sbrl) [inline pages/linux/ndctl.md]: Perhaps ....`on the bus`..... would read better?

**Comment by stellarhopper** [comment]
@sbrl Thanks for the comments, I agree with all of them, I'll send an update.

File: pages/linux/ndctl.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# ndctl
+
+> Utility for managing Non-Volatile DIMMs.
```
[the remark this answers is not in the document]
Comment (stellarhopper) [reply pages/linux/ndctl.md]: They're often abbreviated as NVDIMMs, that's the only reason. But if you prefer lower case, I'm happy to change it.

File: pages/linux/ndctl.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- Change the mode of a namespace to 'raw':
+
+`ndctl create-namespace --reconfigure={{namespaceX.Y}} --mode={{raw}}`
+
+- Check a sector mode namespace for consistency, and repair if needed:
+
+`ndctl check-namespace --repair {{namespaceX.Y}}`
+
+- List all namespaces, regions, buses - including disabled ones:
```
[the remark this answers is not in the document]
Comment (stellarhopper) [reply pages/linux/ndctl.md]: I'll update it to say this:
"List all namespaces, regions, and buses (including disabled ones):"
I think that makes the most sense.

**Comment by sbrl** [comment]
Thanks, @stellarhopper :smiley_cat:

**Pull request merged**